public/app/features/backup/components/StorageLocations/AddStorageLocationModal/AddStorageLocationModal.styles.ts

```diff
@@ -0,0 +1,17 @@
+import { css } from 'emotion';
+
+export const getStyles = () => ({
+  button: css`
+    min-width: 80px;
+    justify-content: center;
+  `,
+  testButton: css`
+    background: linear-gradient(180deg, #04be5b 0%, #04b155 100%);
+    border: 1px solid #058943;
```

Comment:
Also there's a theme variable for this 1px `theme.border.width.sm`

Reply:
Thanks 👍